lib/engine/action/program_auction_bid.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+require_relative 'program_enable'
+
+module Engine
+  module Action
+    class ProgramAuctionBid < ProgramEnable
+      attr_reader :bid_target, :enable_maximum_bid, :maximum_bid, :enable_buy_price, :buy_price, :auto_pass_after
+
+      def initialize(entity, bid_target:, maximum_bid:, buy_price:, enable_maximum_bid: false,
+                     enable_buy_price: false, auto_pass_after: false)
+        super(entity)
```

Comment:
why are you calling super

Reply:
All 6 other "program_..." action classes already defined do it.